Assets/Scripts/Animator/PoseLandmarksRecognizer.cs

```diff
@@ -0,0 +1,89 @@
+// Copyright 2021-2022 The SeedV Lab.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+
+using Mediapipe;
+using Mediapipe.Unity;
+using UnityEngine;
+
+namespace SeedUnityVRKit {
+  public class PoseLandmarksRecognizer {
+    /// <summary>Scale factor to z axis. To be tuned.</summary>
+    private const float Z_SCALE = 0.3f;
```

Comment:
Google style: private const variable should be named `_camelCase` too.

> Naming convention is unaffected by modifiers such as const, static, readonly, etc.

Reply:
Fixed.